src/env.cpp

```diff
@@ -675,6 +675,7 @@ lk::vardata_t *lk::vardata_t::index(size_t idx) const {
 }
 
 lk::vardata_t *lk::vardata_t::lookup(const lk_string &key) const {
+    if (this == 0) throw error_t(lk_tr("access violation: expected hash table, but found null")); // Check nullptr before the next line throws uncatchable errors
```

Comment:
In XCode 14.1 this causes a warning (which is causing the Github Actions to fail):
![image](https://user-images.githubusercontent.com/6498311/199682763-d18baa96-e646-46e1-8dce-33d592181c71.png)

Reply:
Suggest changing to the following that was tested on MacOS 12.6 with XCode 14.1
![image](https://user-images.githubusercontent.com/6498311/199686779-238ae989-acfe-403c-b8ef-4727773139d9.png)